### PR TITLE
Add French translation

### PR DESCRIPTION
**Pull request by forresst**
Last commit from repo ava included in this commit : a9af1da47ef8bab70fc205d09f9906017da55404

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Localized [AVA](https://github.com/sindresorhus/ava) docs
 
-- None yet.
+- [French](fr_FR/readme.md)
```
Comment (sindresorhus) [inline readme.md]: Can you write `French` in french? `Français` or something.

**Comment by sindresorhus** [comment]
Super nice work @forresst :)

What's the recommended way for contributors to keep the localization up to date? Always include the corresponding AVA commit in commit messages and do a git diff? Would be nice to maybe have a contributing.md where we document the best practise.

File: readme.md (hunk 1 above)
Comment (sindresorhus) [inline readme.md]: Also append `by @forresst` at the end ;)

**Comment by vadimdemedes** [comment]
This is awesome! Thank you @forresst!

**Comment by forresst** [comment]
@sindresorhus, I update according your notes.

I agree that we must have a best practice to keep the localization up to date. "Always include the corresponding AVA commit in commit messages" is a good solution, because I did ;-). 

Regarding the `contributing.md` file, the name is a concern, there is already in the translation (the translation of the file `contributing.md` repo `AVA`). I can create a file `contributingtranslation.md` in the repo  `ava-docs`, which can also be translated.
Or second solution, add this in the file `contributing.md` repo `AVA` in a paragraph specifically for translations.

What do you think ? I can do a PR for this.

**Comment by sindresorhus** [comment]
Landed :)

> Regarding the contributing.md file, the name is a concern, there is already in the translation (the translation of the file contributing.md repo AVA). I can create a file contributingtranslation.md in the repo ava-docs, which can also be translated.

There's no contributing.md in the root of the repo currently, so we can put it there. There's no need of translating this contributing.md as it's expected that the translators know English, since that's what they're translating from.

> Or second solution, add this in the file contributing.md repo AVA in a paragraph specifically for translations.

It should stay in this repo as this is where the translation happens.

**Comment by sindresorhus** [comment]
@forresst How should we link to the translations from the main documents. I was thinking something like:

> Translations: Français

Somewhere close to the top. Same in `contributing.md`.

**Comment by forresst** [comment]
@sindresorhus 

> There's no contributing.md in the root of the repo currently, so we can put it there. There's no need of translating this contributing.md as it's expected that the translators know English, since that's what they're translating from.

You are right, I'll create a contributing.md file in English.

> How should we link to the translations from the main documents

Yes to the top. To make all the translations in the same way. For example (_in italics I translated to better understand_). Also we can add the last commit link:

> **Fichier original** (_original file_)
> Traduction du fichier [readme.md](https://github.com/sindresorhus/ava/blob/master/readme.md) (_translation file readme.md_)
> Dernier commit du dépôt de AVA inclus dans cette traduction (_Last commit from repo AVA included this translation_) : https://github.com/sindresorhus/ava/commit/9caa7da2f5e9629be335740ae8f13821a9f3b20e

I will do it in contributing.md and i will add a link for each translation in french.

**Comment by sindresorhus** [comment]
@forresst Added a link from the main readme/contributing.md ;) https://github.com/sindresorhus/ava/commit/8346d35584878dab70318aa37b8a27937c39bcd2

> You are right, I'll create a contributing.md file in English.

Can you open an issue here about that so we don't forget?

**Comment by forresst** [comment]
> Added a link from the main readme/contributing.md ;) sindresorhus/ava@8346d35

@sindresorhus Cool ! We can perhaps do it too for [maintening.md](https://github.com/sindresorhus/ava/blob/master/maintaining.md) file ?

> Can you open an issue here about that so we don't forget?

I did the necessary (#2), there is one week. I hope to provide a first file in early 2016.

**Comment by sindresorhus** [comment]
> We can perhaps do it too for maintening.md file ?

I think we discussed this somewhere else. Can't remember where. There isn't much point in that as `maintaining.md` is for the maintainers and they're expected to know English anyways.

**Comment by forresst** [comment]
> I think we discussed this somewhere else. Can't remember where. There isn't much point in that as maintaining.md is for the maintainers and they're expected to know English anyways.

@sindresorhus This is `maintaining.md` of ava-docs repository which must be and remain in English (see above). `maintaining.md` of ava repository can be translated, I did elsewhere.
But no worries, it's like you want.

Happy New Year :tada:

**Comment by sindresorhus** [comment]
> maintaining.md of ava repository can be translated

But there's no point. That file is for the AVA team which is expected to know English.

**Pull request closed without merging**